docs/source/dataset_script.mdx

```diff
@@ -270,85 +270,27 @@ def _generate_examples(self, filepath):
 ## Testing data and checksum metadata
 
 We strongly recommend adding testing data and checksum metadata to your dataset to verify and test its behavior. This ensures the generated dataset matches your expectations.
```

Comment:
I think we can remove all the mentions of "testing data"

Reply:
I agree we should remove all mentions to testing data.